lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3296,7 +3301,11 @@ namespace {
       for (auto member : members) {
         result->addMember(member);
       }
-      
+
+      for (auto method : methods) {
+        result->addMember(method);
+      }
+
```

Comment:
I don't know if this actually makes any difference, but could you add these after the ctors?  I think that's the style-canonical order.